### PR TITLE
Gaussian: move formatted checkpoint files to testing directory

### PR DESCRIPTION
**Pull request by berquist**
This moves the Gaussian formatted checkpoint outputs under the `data/FChk` directory and starts testing them.

### CODE REVIEW

File: test/data/testTD.py [hunk 1]
```diff
@@ -164,10 +169,6 @@ class GaussianTDDFTTest(GenericTDTest):
 
     expected_l_max = 48000
 
-    def testrotatsnumber(self):
```
Comment (berquist) [inline test/data/testTD.py:167]: This is duplicated from the parent test class.

**Comment by berquist** [comment]
I think it can still use things for `moments`...

**Comment by berquist** [comment]
A bunch of Horton bridge stuff is broken.  I am not sure how this code wasn't being tested before.

**Comment by berquist** [comment]
Moments can wait, this is too big already.

**Review by shivupa — APPROVED** [review]

**Pull request merged**